src/WebAppDIRAC/Lib/WebHandler.py

```diff
@@ -357,7 +357,7 @@ def __checkPath(self, setup, group, route):
 
   def get(self, setup, group, route):
     if not self._pathResult.ok:
-      raise self._pathResult
+      raise Exception(self._pathResult)
```

Comment:
I'm less sure about these two changes as I can't figure out what catches the exceptions.

@TaykYoku Any ideas?

Reply:
In this place self._pathResult  is `WErr` that child of the `tornado.web.HTTPError`. `Exception(self._pathResult)` will also work but the output will change.
What is the problem with using WErr?

Reply:
Ah right thanks, I understand the actual issue now. The issue was a `pylint` saying:
```python
************* Module src.WebAppDIRAC.Lib.WebHandler
src/WebAppDIRAC/Lib/WebHandler.py:357:6: E0710: Raising a new style class which doesn't inherit from BaseException (raising-non-exception)
src/WebAppDIRAC/Lib/WebHandler.py:397:6: E0710: Raising a new style class which doesn't inherit from BaseException (raising-non-exception)
```
It's debatable if this is a false-positive  or not in the general case but for DIRAC it definitely is.

That said it seems that `WOK` is basically unused so I think it can just be removed. Does 0fe8bc7 look reasonable to you?

Reply:
Yes, 0fe8bc7 looks good for me